test/jdk/java/util/Properties/PropertiesStoreTest.java

```diff
@@ -91,6 +97,26 @@
         };
     }
 
+    /**
+     * Returns a {@link Locale} to use for testing
+     */
+    @DataProvider(name = "localeProvider")
+    private Object[][] provideLocales() {
+        // pick a non-english locale for testing
+        Set<Locale> locales = Arrays.stream(Locale.getAvailableLocales())
+                .filter(l -> !l.getLanguage().isEmpty() && !l.getLanguage().equals("en"))
+                .limit(1)
+                .collect(Collectors.toSet());
+        locales.add(Locale.getDefault()); // always test the default locale
+        locales.add(Locale.US); // guaranteed to be present
+        locales.add(Locale.ROOT); // guaranteed to be present
```

Comment:
Can we assume the returned `Set<Locale>` is mutable? `Collectors.toSet()` javadoc reads no guarantee for mutability.

Reply:
That's a very good point. I've updated the PR to now explicitly use a mutable `Set` instead of using `Collectors.toSet()`

Reply:
This is ok, although `Collectors.toCollection(HashSet::new)` is a bit concise.

Reply:
Hello Naoto, I've updated the PR to use `HashSet::new`.